org/allPRs.ts

```diff
@@ -51,6 +51,21 @@ export const xcodeprojConfiguration = async () => {
     }
 }
 
+export const localizedStrings = async () => {
+    for (let file of danger.git.modified_files) {
+        let diff = await danger.git.diffForFile(file);
+        let addedLines = diff?.added.split(/\n/);
+        // The regex is equal to:
+        // * word boundary
+        // * NSLocalizedString(
+        // This way it will match `NSLocalizedString(` but not `NSLocalizedString` (without the opening parenthesis, which could be used in a comment).
+        if (addedLines?.find(value => /\bNSLocalizedString\(/.test(value))) {
+            message("You seem to be updating localized strings. Make sure that you request translations and include translated strings before you ship your change.");
```

Comment:
🔍 What do you think about adding a link to the translation guidelines: https://app.asana.com/0/904401899170/1206727265537758/f?

Reply:
Absolutely! Let me add it